packages/amplication-client/src/Components/TopicNameField.tsx

```diff
@@ -41,7 +41,6 @@ const TopicNameField = ({ name, ...rest }: Props) => {
       <TextInput
         {...field}
         {...rest}
-        label="Name"
```

Comment:
this can stay "Name"

Reply:
The label property is deconstructed from rest: `...rest` <- label is in here.